compiler/rustc_hir/src/hir_id.rs

```diff
@@ -62,69 +61,3 @@ pub const CRATE_HIR_ID: HirId = HirId {
     owner: LocalDefId { local_def_index: CRATE_DEF_INDEX },
     local_id: ItemLocalId::from_u32(0),
 };
-
-#[derive(Clone, Default, Debug, Encodable, Decodable)]
-pub struct HirIdVec<T> {
```

Comment:
This has been introduced by #79519 and ended up not being used in that PR.
However, I plan to use it in #72015 and related PRs.

Reply:
Ok, I added it back with a comment.